__tests__/api/friends/accept/route.test.ts

```diff
@@ -0,0 +1,211 @@
+import { POST } from "@/app/api/friends/accept/route";
+import myGetServerSession from "@/lib/myGetServerSession";
+import fetchRedis from "@/helpers/redis";
+import { db } from '@/lib/db';
+
+jest.mock("../../../../src/lib/myGetServerSession",()=>({
```

Comment:
mocking the absoulte path is darn tricky. plus it looks dumb. Is there a way to get jest to recognize the '@/' alias the way next.js regular does?